Domain Connect Spec Draft.adoc

```diff
@@ -438,32 +444,30 @@ This is the URL used to ask for consent and to apply a template to a
 domain. It is called from the Service Provider to start the Domain
 Connect Protocol.
 
-This URL can be called in two ways.
+This URL can be called in one of two ways. 
+
+===== New Broswer Window
 
 The first is through a new browser
 tab or in a popup browser window. The DNS Provider would sign the user
 in if necessary, verify domain ownership, and ask for confirmation
 before application of the template. After application of the template,
 the DNS Provider would automatically close the browser tab or window.
 
+===== Same Browser Window
 The second is in the current browser tab/window. As above the DNS
 Provider would sign the user in if necessary, verify domain ownership,
 and ask for confirmation before application of the template. After
 application of the template (or cancellation by the user), the DNS

```

Comment:
Just clarifying when the user cancels there will be an error sent back. This is to make sure third paragraph in section 4.2.5 is accurate.

Reply:
I see the edit at line 500 on error description for this scenario, but returning an error is mandatory for this scenario. Otherwise it invalidates section 4.2.5 paragraph three.